Makefile

```diff
@@ -359,6 +359,14 @@ octant-antrea-ubuntu:
 	docker tag antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION) projects.registry.vmware.com/antrea/octant-antrea-ubuntu
 	docker tag antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION) projects.registry.vmware.com/antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION)
 
+.PHONY: antrea-mc-controller-ubuntu
+antrea-mc-controller-ubuntu:
+	@echo "===> Building antrea/antrea-mc-controller-ubuntu Docker image <==="
```

Comment:
Why not consider no `--pull` docker build? Maybe someone wants to build locally.

Reply:
sure, I will add a condition check to build it without  `--pull`